src/modules/package.rs

```diff
@@ -960,6 +975,17 @@ java {
         expect_output(&project_dir, None, None);
         project_dir.close()
     }
+    #[test]
+    fn test_extract_grade_version_from_properties() -> io::Result<()> {
+        let config_name = "gradle.properties";
+        let config_content = "
+            version=1.2.3
+            ";
+        let project_dir = create_project_dir()?;
+        fill_config(&project_dir, config_name, Some(config_content))?;
+        expect_output(&project_dir, None, None);
```

Comment:
The test should show that the version number can be extracted:
```suggestion
        expect_output(&project_dir, Some("v1.2.3"), None);
```